app.js

```diff
@@ -117,14 +95,15 @@ Array.prototype._some = function(callbackFunction) {
 	return false;
 };
 
-function _every(array, callbackFunction) {
-	_forEach(array, element => {
+//este metodo NO anda
```

Comment:
Array.prototype._every = function(callbackFunction) {
	for (const element of this) {
		if (!callbackFunction(element)) {
			return false;
		}
	}
	return true;
};